servers/rendering/renderer_rd/cluster_builder_rd.cpp

```diff
@@ -419,7 +419,7 @@ void ClusterBuilderRD::begin(const Transform3D &p_view_transform, const Projecti
 void ClusterBuilderRD::bake_cluster() {
 	RENDER_TIMESTAMP("> Bake 3D Cluster");
 
-	RD::get_singleton()->draw_command_begin_label("Bake Light Cluster");
+	RD::get_singleton()->draw_command_begin_label(Span<char>("Bake Light Cluster"));
```

Comment:
Explicit conversion to `Span<char>` shouldn't be required for `"string"` literals. You should be able to save all these line changes :)

Reply:
Unfortunately the `draw_command_begin_label` is ambiguous because of `String` being the alternative, so didn't have a choice

Reply:
Oh, right, that's still a problem.
It should be solvable if the `String` version instead accepts a `Span<char32_t>`, although that will probably nuke the bindings. I guess what you have now is fine until we figure out a better solution.